pkg/notes/document/document.go

```diff
@@ -326,70 +322,49 @@ func sortKinds(notesByKind NotesByKind) []Kind {
 	return res
 }
 
+// urlPrefixForBucket returns the URL prefix for the provided bucket string
+func urlPrefixForBucket(bucket string) string {
```

Comment:
We're going to need something like this for anago.
Can you move this to the release pkg instead?

Reply:
Yes